styles/styles.css

```
@@ -57,29 +57,163 @@ body {
   font-size: 16px;
   font-weight: 700;
   line-height: 20px;
+  text-decoration: none;
   color: #2d3440;
-  border-radius: 16px;
+  border-radius: 26px;
+  outline: 4px solid rgb(247, 202, 214);
   background-color: #fcfcfc;
+  box-shadow: 0 4px 12px 0 rgba(45, 52, 64, .10);
+}
+
+.button-settings {
+  box-sizing: border-box;
+  padding: 8px 32px;
+  font-size: 14px;
+  font-weight: 700;
+  border: none;
+  border-radius: 20px;
+  outline: 2px solid #fcfcfc;
+  background: rgba(252, 252, 252, .30);
+}
+
+.button-settings:hover {
+  box-sizing: border-box;
+  border: none;
+  background: #fcfcfc;
+}
+
+.button-settings:active {
+  box-sizing: border-box;
+  border: none;
+  background: rgba(252, 252, 252, .50);
+}
+
+.button-settings:focus {
+  box-sizing: border-box;
+  border: none;
+  outline: 2px solid var(--basic-extra-dark, #2d3440);
+  background: rgba(252, 252, 252, .30);
```

Прописывать те же самые свойства нет смысла - они и так применяются (например `box-sizing: border-box;` и так задан в "обычном" состоянии и прописывать его в каждом нет смысла). А вот что нужно прописывать - это меняющиеся свойства. То есть если у нас при наведении меняется только `background` - только его и оставляем

По поводу  `var(--basic-extra-dark, #2d3440)`. `var` в css это использование переменных, и для того чтобы ими пользоваться они должны быть объявлены, например в `:root`. А второй параметр - это фолбек на случай когда нет такой переменной

```css
:root {
  --basic-extra-dark: #2d3440;
}

...

.button-settings:focus {
  outline: 2px solid var(--basic-extra-dark);
}
```

Вот в таком случае имеет смысл все это прописывать. А у нас нет переменных в проекте, мы их не объявляли и тд. И в таком случае у нас **всегда** отрабатывает фолбечный вариант - зачем прописывать лишнее?

```suggestion
.button-settings:hover {
  background: #fcfcfc;
}

.button-settings:active {
  background: rgba(252, 252, 252, .50);
}

.button-settings:focus {
  outline: 2px solid #2d3440;
```